### ISSUE TITLE
error while loading shared libraries: libopencv_imgcodecs3.so.3.1

### ISSUE THREAD

**Issue by mzaiady** [nmellado/Super4PCS#50]
/install/scripts$ ./run-example.sh

```
  ------------------------------
Running Super4PCS
../bin/Super4PCS: error while loading shared libraries: libopencv_imgcodecs3.so.3.1: cannot open shared object file: No such file or directory
Command exited with non-zero status 127
real 0.00
user 0.00
sys 0.00
   ------------------------------

```

**Comment by nmellado** [comment]
Dear @mzaiady,

Thanks for your kind request.
Despite many careful reads and a lot of effort I couldn't figure out your operating system, compiler, compiling options, or installed libraries. 
Even worst, I couldn't find any question in your message.

This sadly prevents me to help you, and I'm sorry to inform you that I'm closing this issue, as you failed to comply to basic rules of courtesy.

Best wishes,

N.

**Comment by mzaiady** [comment]
Thank you for replay, sorry for missing information!

Installation and build works fine, but I’m getting the error while running the ./run-example.sh
I'm planning to use it for within ROS, PCL 1.7

Current configuration:

OS: Ubuntu 14.0.4

from CMakeCache.txt

OpenCV_3RDPARTY_LIB_DIR_DBG:PATH=/opt/ros/indigo/share/OpenCV-3.1.0-dev/3rdparty/lib
OpenCV_3RDPARTY_LIB_DIR_OPT:PATH=/opt/ros/indigo/share/OpenCV-3.1.0-dev/3rdparty/lib
OpenCV_CONFIG_PATH:FILEPATH=/opt/ros/indigo/share/OpenCV-3.1.0-dev
OpenCV_DIR:PATH=/opt/ros/indigo/share/OpenCV-3.1.0-dev

**Comment by nmellado** [comment]
Hi,

Thanks for the report.

So, if I'm right, OpenCV libraries are not included in your path, right ?
One fix would be to add the OpenCV library path to the applications RPATH. 

I can try to push a tentative fix tomorrow. In the meantime, adding OpenCV libraries either in your system library dir, or in Super4PCS lib folder should temporarly fix the problem.

Cheers

**Comment by nmellado** [comment]
Hi,

I've pushed a candidate fix in #51 .
Could you please checkout branch `fix_rpath_from_master` and check if solves your problem ?

Thanks

**Comment by mzaiady** [comment]
It Works Fins now, thank you very much!

Additional query, maybe should open another issue! I have PCL 1.7 Would  PCLWrapper work with it!
